editor/plugins/script_text_editor.cpp

```diff
@@ -238,10 +238,6 @@ void ScriptTextEditor::_show_warnings_panel(bool p_show) {
 void ScriptTextEditor::_warning_clicked(Variant p_line) {
 	if (p_line.get_type() == Variant::INT) {
 		goto_line_centered(p_line.operator int64_t());
-	} else if (p_line.get_type() == Variant::DICTIONARY) {
-		Dictionary meta = p_line.operator Dictionary();
-		code_editor->get_text_editor()->insert_line_at(meta["line"].operator int64_t() - 1, "# warning-ignore:" + meta["code"].operator String());
```

Comment:
Couldn't it be kept by adding something like `vformat("@warning_ignore(%s)", meta["code"].operator String())`?

Reply:
The problem is that this ignores indentation. It wasn't a problem for comments but it is for annotations. So a proper solution is a bit more complicated.